t/conn.t

```diff
@@ -114,23 +114,23 @@ committed: false
 3: 0, conn: 1
 committed: true
 4: 0, conn: 2
-committed: false
+committed: true
 5: 0, conn: 2
 committed: true
 6: 1, conn: 3
-committed: false
+committed: true
```

Comment:
The changes do not follow the document.
https://github.com/openresty/lua-resty-limit-traffic/pull/80/files#diff-aa64cd78441de2bae2f552bca126b6cedde9a9fa7ba046d689386cbffecc5795R229

Reply:
Well, I doubt if this change makes sense for `is_committed()` docs:

```diff
- Returns `true` if the previous [incoming](#incoming) call actually commits the event
+ Returns `true` if the previous [incoming](#incoming) calls actually have committed the event
```